src/FirstSteps/RPG/Heroes/Dwarf.cs

```diff
@@ -22,5 +23,16 @@ public string Greed()
         {
             return "\nYou are playing as a Dwarf\n";
         }
+        public override int DealDamage()
+        {
+            Item item = equipment.backpack.FirstOrDefault(
```

Comment:
You can use [Any](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.any?view=net-8.0#system-linq-enumerable-any-1(system-collections-generic-ienumerable((-0))-system-func((-0-system-boolean)))) and check if any item in equipment is `MagixAxe` type. Then you don't have to use [magic strings](https://en.wikipedia.org/wiki/Magic_string).